docs/UserGuide.md

```diff
@@ -18,85 +18,107 @@ title: User Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 2. About
+## 2. How to use this User Guide
 
-### 2.1 Using this User Guide
+Welcome to *_MyInsuRec_ User Guide*!
 
-Welcome to *MyInsuRec User Guide*!
+Our team would like to thank you for choosing to use _MyInsuRec_! <br>
```

Comment:
maybe thank you for choosing _MyInsuRec_ will be better?

Reply:
i think thomas suggestion is better for this, since i think our team isn't introduced in this UG

Reply:
Okay will make this change later

Reply:
Okay will make this change later